### DISCUSSION TITLE
Welcome to lwgps Discussions!

### DISCUSSION THREAD

**Discussion by MaJerle** [MaJerle/lwgps#18] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by uladzimir-simankou** [comment]
Hi Tilen. Could you clarify please why lat/lon terms are not parsed in RMC parser?
if to take the first sentence from example.c 
"$GPRMC,183729,A,3907.356,N,12102.482,W,000.0,360.0,080301,015.5,E*6F\r\n"
and feed to lwgps_process then position and utc time stay 0 although they are present in the sentence.

hgps	lwgps_t	{...}	
	latitude	lwgps_float_t	0	
	longitude	lwgps_float_t	0	
	altitude	lwgps_float_t	0	
	geo_sep	lwgps_float_t	0	
	sats_in_use	uint8_t	0 '\0'	
	fix	uint8_t	0 '\0'	
	hours	uint8_t	0 '\0'	
	minutes	uint8_t	0 '\0'	
	seconds	uint8_t	0 '\0'	
	dop_h	lwgps_float_t	0	
	dop_v	lwgps_float_t	0	
	dop_p	lwgps_float_t	0	
	fix_mode	uint8_t	0 '\0'	
	satellites_ids	uint8_t [12]	0x555555559081 <hgps+65>	
	sats_in_view	uint8_t	0 '\0'	
	is_valid	uint8_t	1 '\001'	
	speed	lwgps_float_t	0	
	course	lwgps_float_t	360	
	variation	lwgps_float_t	15.5	
	date	uint8_t	8 '\b'	
	month	uint8_t	3 '\003'	
	year	uint8_t	1 '\001'	
	p	struct {...}	{...}	
		stat	lwgps_statement_t	STAT_RMC	
		term_str	char [13]	0x5555555590b4 <hgps+116>	
		term_pos	uint8_t	3 '\003'	
		term_num	uint8_t	12 '\f'	
		star	uint8_t	1 '\001'	
		crc_calc	uint8_t	111 'o'	
		data	union {...}	{...}	
		
Could you explain what is the point please?

**Reply by MaJerle** [reply]
Global position data is parsed from GPGGA statement instead.

**Reply by uladzimir-simankou** [reply]
I have not worked with the real sensor yet. The way I see it is to configure chip on periodic send of RMC and disable other formats. Since RMC has everything needed: time, position and is position valid flag. So to save CPU resources it makes sense to read one sentence instead of two. But I'm not sure if it is possible on a real hardware though.
